run.py

```diff
@@ -28,24 +29,28 @@ def write_topics(repo, topiclist, dry_run):
 g = Github(token)
 
 try:
-    user_repositories = list(g.get_user().get_repos())
+    print("Getting list of repositories")
+    if args.organization:
+        repositories = [r for r in g.get_user().get_repos(affiliation = "organization_member", visibility = "public") if r.organization.login == args.organization]
+    else:
+        repositories = list(g.get_user().get_repos(affiliation = "owner", visibility = "public"))
     user_id = g.get_user().id
 except BadCredentialsException:
     print("Token is invalid or has the wrong permissions")
     exit(1)
 
-for repo in user_repositories:
+for repo in repositories:
     print(f"> Checking {repo.full_name}:")
 
     if repo.private or repo.fork or repo.archived:
```

Comment:
At least the `repo.private` check shouldn't be neccessary anymore due to the new filters in `get_repos()` I guess?

Reply:
Yes, I'll remove that one.